176 alpha vantage

pteetor opened the pull request:
I rebased this branch after incorporating Josh's suggested edits. I believe it is ready for the merge to master.

joshuaulrich commented:
Thanks Paul!  The PR contained a merge commit, which I rebased and removed.  I also tweaked the commit message a bit.

pteetor commented:
Excellent! Thank you for your help and patience, Josh. You introduced me to
*rebase*; never used it before because, hey, why would I erase all those
poetic commit messages? I understand now why my *rebase* mistakenly
included the merge commit (oops).

And I gotta get into the habit of referencing back to an Issue inside my
commit message (thank you). At Wm Blair, our project is transitioning from
the "cowboy coding" of a research project to the "corporate coding" of an
audit-proof financial project. Hello, Issue Tracking.

On Sat, Sep 23, 2017 at 9:44 PM, Joshua Ulrich <notifications@github.com>
wrote:

> Thanks Paul! The PR contained a merge commit, which I rebased and removed.
> I also tweaked the commit message a bit.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/joshuaulrich/quantmod/pull/189#issuecomment-331682954>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABH0ca2rUpTuD14Us1dPc2Ud6oxUYanpks5slcIegaJpZM4PTBk->
> .
>



-- 
Paul Teetor
Elgin, IL, USA

http://quantdevel.com/public/
